### ISSUE TITLE
VS Code Update Failure due to code.exe in use by another process

### ISSUE THREAD

**Issue by podo12** [microsoft/vscode#85499]
- VSCode Version:
- OS Version:

Steps to Reproduce:

1.
2.

Does this issue occur when all extensions are disabled?: Yes/No

**Comment by podo12** [comment]
Just a head's up...I've been searching for this answer for weeks and finally stumbled on the fix.  Since every issue on this seemed to be marked closed with no answer, I'm just posting this in case someone else is as frustrated as me...

VS Code kept trying to update and would pop-up every 10 minutes or so.  Each time I tried to update it would fail due to "The process cannot access the file because it is being used by another process".

One hint I found was that the issue was connected to One Drive, so I stopped One Drive and continued.  It corrupted my VS Code forcing me to reinstall.

When I went to uninstall VS Code to start over it said the elements could not be uninstalled and would need to be removed manually.  I found the install folder at: users\...\AppData\Local\Programs\Microsoft VS Code.  The folder appeared untouched by the uninstall process and still contained the code.exe file.

When I tried to delete the folder it said that files were in use by another process.   

I had already closed down OneDrive, so I hunted around and found Skype running in the background (I've never used it...).  I shut down Skype and was immediately able to delete the folder.

I re-installed VS Code (and uninstalled Skype to try to avoid future issues).  When I ran VS Code, it opened up exactly as I had left it, then about a minute in, it wanted to update again. 

I hit update now and got the same error about it being in use by another process.

Since I had rebooted several times, OneDrive was running again so I closed it and hit retry.

The update worked fine and all appears normal now.

Hope this helps someone...
